build/python/build/libs.py

```diff
@@ -53,13 +52,6 @@
     use_actual_arch=True,
 )
 
-openssl = OpenSSLProject(
```

Comment:
Why?

Reply:
Because its alternative, wolfssl is used. Is it used in somewhere? You want to keep it?

Reply:
It wasn't used previously, you didn't disable its use, so this is irrelevant for this PR. You may suggest removing it because it's unused, but not hidden inside an unrelated commit in an unrelated PR.

Reply:
Will resume openssl.